parm/config/config.prep

```diff
@@ -17,6 +18,8 @@ export DO_RELOCATE="NO"
 export TROPCYQCRELOSH="$HOMEgfs/scripts/exglobal_atmos_tropcy_qc_reloc.sh"
 export SENDCOM=YES
 
+export COMPONENT=${COMPONENT:-atmos}
```

Comment:
Is this really something that will be set from above?
Isn't component here "always" `atmos`?